library/src/main/java/com/chuckerteam/chucker/internal/data/entity/HttpTransaction.kt

```diff
@@ -214,4 +214,37 @@ internal class HttpTransaction(
         scheme = url.scheme()
         return this
     }
+
+    // Not relying on 'equals' because comparison be long due to request and response sizes
+    // and it would be unwise to do this every time 'equals' is called.
+    @Suppress("ComplexMethod")
+    fun hasTheSameContent(other: HttpTransaction?): Boolean {
+        if (this === other) return true
+        if (other == null) return false
+
+        return id == other.id &&
```

Comment:
```suggestion
        return (id == other.id) &&
```

Reply:
Could you, please, add braces for every comparison to make it more readable?

Reply:
Sure.

Reply:
Should I enable a Deketk/ktLint rule for this?

Reply:
@cortinico I think it would be helpful if that's the way to go with conditions. One other thing I'd like to see is added `codeStyleConfig` to repository becuase sometimes I have to adjust stuff like line length or indentation manually as IDE does not format it for me properly and it is only caught by Detekt.

Reply:
I can create a separate issue for this. Or not if you do not want `.idea` stuff in repository. :)

Reply:
Good idea, actually. Also struggling with this styling issue all the time, since got used to use auto format feature a lot.

Reply:
Creating an issue, so we could track it.